### ISSUE TITLE
pip install dependency errors

### ISSUE THREAD

**Issue by spestana** [DSHydro/PyPlanetSCA-Python-Library#16]
I wanted to re-install the package to continue development and testing with the latest version, so I started fresh with a brand new conda environment, but received error when trying to install from test pypi.

The error messages have suggestions on how to fix, regarding huggingface-hub package versions (see the links they provided).

Steps to reproduce errors (running on [CryoCloud](https://hub.cryointhecloud.com)):
1. Create new conda environment `conda create --name planetenv`
2. Install from test pypi `pip install -i https://test.pypi.org/simple/ planetsca`
3. Received the following error:
```
(planetenv) jovyan@jupyter-spestana:~/git/planetsca$ pip install -i https://test.pypi.org/simple/ planetsca
Looking in indexes: https://test.pypi.org/simple/
Collecting planetsca
  Downloading https://test-files.pythonhosted.org/packages/5c/54/18ebeeec7f215e053ff6daf3e204458bdc26be28221777fe203729073836/planetsca-0.1.dev111-py3-none-any.whl.metadata (6.4 kB)
Collecting dvc>=2.10.0 (from planetsca)
  Using cached https://test-files.pythonhosted.org/packages/22/ea/e2969794a37ea2958471fad368e918daf740e3f6e375d19ed04f691e0cab/dvc-3.51.2-py3-none-any.whl.metadata (17 kB)
Requirement already satisfied: geopandas in /srv/conda/envs/notebook/lib/python3.11/site-packages (from planetsca) (0.13.2)
INFO: pip is looking at multiple versions of planetsca to determine which version is compatible with other requirements. This could take a while.
Collecting planetsca
  Downloading https://test-files.pythonhosted.org/packages/74/06/dc2250f75a4dccb01fc134e6d9eeadafc7c4bd58c7b820ad9c18bb1aeb50/planetsca-0.1.dev110-py3-none-any.whl.metadata (6.3 kB)
  Downloading https://test-files.pythonhosted.org/packages/18/d9/42339262b3fae3d2e4de9f229ff9cab558c235e2578f9b68098c36e74184/planetsca-0.1.dev97-py3-none-any.whl.metadata (6.3 kB)
  Downloading https://test-files.pythonhosted.org/packages/ac/ab/e42870b71b5b720fdaba2b24aa29fbb3a1117b60893060bf3c422d086121/planetsca-0.1.dev95-py3-none-any.whl.metadata (6.3 kB)
ERROR: Cannot install planetsca==0.1.dev110, planetsca==0.1.dev111, planetsca==0.1.dev95 and planetsca==0.1.dev97 because these package versions have conflicting dependencies.

The conflict is caused by:
    planetsca 0.1.dev111 depends on huggingface-hub>=0.23.2
    planetsca 0.1.dev110 depends on huggingface-hub>=0.23.2
    planetsca 0.1.dev97 depends on huggingface-hub>=0.23.2
    planetsca 0.1.dev95 depends on huggingface-hub>=0.23.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

**Comment by IanChiu-333** [comment]
This is the same issue as #18 pretty sure. For #18 it's displaying two errors when it looks into the single version and this is displaying the same errors just for multiple versions.

**Comment by spestana** [comment]
Solved (thanks again @lsetiawan !):

When we tried a pip install this way, pip sees that planetsca is part of the test.pypi.org package index, and therefore searches for all dependencies within the same package index. The dependencies that we want, however, are in the production pypi.org package index.

Therefore we have to tell pip to install planetsca from test.pypi.org, but let it use the default index (pypi.org) for all dependencies.

This can be done with:
`pip install --extra-index-url https://test.pypi.org/simple planetsca`
